src/index.html

```diff
@@ -9,6 +9,38 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="http://" class="logo">
+        <img src="images/logo.png" alt="Main">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__link">
+            <a href="./index.html" class="link is-active">Apple</a>
+          </li>
+          <li class="nav__link">
+            <a href="http://" class="link">Samsung</a>
```

Comment:
You should create anchor links
```suggestion
            <a href="#samsung" class="link">Samsung</a>
```